packages/flutter/lib/src/widgets/focus_traversal.dart

```diff
@@ -84,8 +84,45 @@ enum TraversalDirection {
   left,
 }
 
-/// An object used to specify a focus traversal policy used for configuring a
-/// [FocusTraversalGroup] widget.
+/// {@template flutter.focus.TraversalEdgeBehavior}
```

Comment:
Is it really worth having a template for this one sentence? Seems like overkill, and since it's the summary sentence, obscures the summary at the `@macro` sites when just reading the comments in the code.

Reply:
Done.